src/components/views/elements/Pill.js

```diff
@@ -261,7 +261,7 @@ const Pill = createReactClass({
         }
 
         const classes = classNames("mx_Pill", pillClass, {
-            "mx_UserPill_me": userId === MatrixClientPeg.get().credentials.userId,
+            "mx_UserPill_me": userId === MatrixClientPeg.get().getUserId(),
```

Comment:
This is the second unrelated change in this PR - please try to keep these out for future PRs. Personally I don't mind having one vaguely related thing in the PR, but after a couple it starts to feel like false advertising on the PR.

Reply:
The change is not unrelated, it was to simplify stubbing for the test, any and all changes in this PR are related either to the change or testing it.

https://github.com/matrix-org/matrix-react-sdk/pull/3711/files#diff-c2d1eb236f1e50980bd094446ee76c63R132

Reply:
things like this should be pointed out so tired eyes don't miss them :(